e2e/features/case-search.feature

```diff
@@ -0,0 +1,106 @@
+    And I should see the option `Bradford` for the `Hearing centre` field
+    And I should see the option `North Shields` for the `Hearing centre` field
+    And I should see the option `Birmingham` for the `Hearing centre` field
+    And I should see the option `Hatton Cross` for the `Hearing centre` field
+    And I should see the option `Glasgow` for the `Hearing centre` field
+    And I should see the `HMCTS reference` field
+    And I should see the `Appeal start date` field
+    And I should see the `Appellant name` field
+
+  @case-search @RIA-1478
+  Scenario: Case search contains correct filters for Case Officer
+
+    Given I am signed in as a `Case Officer`
+    When I click the `Search` link
+    And I should see the option `Immigration & Asylum` for the `Jurisdiction*` field
+    And I should see the `Jurisdiction*` field
+    And I should see the `Case type*` field
+    And I should not see the `State` field
+    And I should see the `Hearing centre` field
+    And I should see the option `Manchester` for the `Hearing centre` field
+    And I should see the option `Taylor House` for the `Hearing centre` field
+    And I should see the option `Newport` for the `Hearing centre` field
+    And I should see the option `Bradford` for the `Hearing centre` field
+    And I should see the option `North Shields` for the `Hearing centre` field
+    And I should see the option `Birmingham` for the `Hearing centre` field
+    And I should see the option `Hatton Cross` for the `Hearing centre` field
+    And I should see the option `Glasgow` for the `Hearing centre` field
+    And I should see the `HMCTS reference` field
+    And I should see the `Appeal start date` field
+    And I should see the `Appellant name` field
+
+  @case-search @RIA-1478
+  Scenario: Case search contains correct filters for Admin Officer
+
+    Given I am signed in as a `Admin Officer`
+    When I click the `Search` link
+    And I should see the option `Immigration & Asylum` for the `Jurisdiction*` field
+    And I should see the `Jurisdiction*` field
+    And I should see the `Case type*` field
+    And I should not see the `State` field
+    And I should see the `Hearing centre` field
+    And I should see the option `Manchester` for the `Hearing centre` field
+    And I should see the option `Taylor House` for the `Hearing centre` field
+    And I should see the option `Newport` for the `Hearing centre` field
+    And I should see the option `Bradford` for the `Hearing centre` field
+    And I should see the option `North Shields` for the `Hearing centre` field
+    And I should see the option `Birmingham` for the `Hearing centre` field
+    And I should see the option `Hatton Cross` for the `Hearing centre` field
+    And I should see the option `Glasgow` for the `Hearing centre` field
+    And I should see the `HMCTS reference` field
+    And I should see the `Appeal start date` field
+    And I should see the `Appellant name` field
```

Comment:
As the story says all users, can we have the HO Users scenario

Reply:
As discussed added a case for a HO user